source/common/local_reply/local_reply.cc

```diff
@@ -0,0 +1,161 @@
+#include "common/local_reply/local_reply.h"
+
+#include <string>
+#include <vector>
+
+#include "common/access_log/access_log_formatter.h"
+#include "common/access_log/access_log_impl.h"
+#include "common/common/enum_to_int.h"
+#include "common/common/substitution_format_string.h"
+#include "common/config/datasource.h"
+#include "common/http/header_map_impl.h"
+
+namespace Envoy {
+namespace LocalReply {
+
+class BodyFormatter {
+public:
+  BodyFormatter() { formatter_ = std::make_unique<Envoy::AccessLog::FormatterImpl>("%RESP_BODY%"); }
```

Comment:
Related to my other comment about how to not leak this into the access logger impl where it is not supported?

Reply:
Done